firewatch-base-configs/opp/lp-interop-aws.json

```
@@ -0,0 +1,26 @@
+      {"step": "acm-tests-clc-create", "failure_type": "all", "classification": "CLC testcase failure", "jira_project": "!default", "jira_component": ["!default"], "group": {"name": "opp-tests", "priority": 4}, "jira_epic": "!default", "jira_assignee": "!default"},
+      {"step": "acm-tests-clc-destroy", "failure_type": "all", "classification": "CLC testcase failure", "jira_project": "!default", "jira_component": ["!default"], "group": {"name": "opp-tests", "priority": 4}, "jira_epic": "!default", "jira_assignee": "!default"},
+      {"step": "interop-tests-ocs-tests", "failure_type": "all", "classification": "Odf testcase failure", "jira_project": "!default", "jira_component": ["!default"], "group": {"name": "opp-tests", "priority": 5}, "jira_epic": "!default", "jira_assignee": "!default"},
+      {"step": "acm-tests-obs", "failure_type": "all", "classification": "OBS testcase failure", "jira_project": "!default", "jira_component": ["!default"], "group": {"name": "opp-tests", "priority": 5}, "jira_epic": "!default", "jira_assignee": "!default"},
+      {"step": "acm-tests-grc", "failure_type": "all", "classification": "GRC testcase failure", "jira_project": "!default", "jira_component": ["!default"], "group": {"name": "opp-tests", "priority": 5}, "jira_epic": "!default", "jira_assignee": "!default"},
+      {"step": "acm-tests-alc", "failure_type": "all", "classification": "ALC testcase failure", "jira_project": "!default", "jira_component": ["!default"], "group": {"name": "opp-tests", "priority": 5}, "jira_epic": "!default", "jira_assignee": "!default"},
+      {"step": "cucushift-installer-check-cluster-health", "failure_type": "all", "classification": "Cucushift testcase failure", "jira_project": "!default", "jira_component": ["!default"], "group": {"name": "opp-tests", "priority": 5}, "jira_epic": "!default", "jira_assignee": "!default"},
+      {"step": "quay-tests-quay-interop-test", "failure_type": "all", "classification": "Quay testcase failure", "jira_project": "!default", "jira_component": ["!default"], "group": {"name": "opp-tests", "priority": 5}, "jira_epic": "!default", "jira_assignee": "!default"},
+      {"step": "openshift-extended-test", "failure_type": "pod_failure", "classification": "OCP Observability failure", "jira_project": "!default", "jira_component": ["!default"], "group": {"name": "opp-tests", "priority": 5, "jira_epic": "!default", "jira_assignee": "!default"}},
+      {"step": "acm-must-gather", "failure_type": "pod_failure", "classification": "ACM must-gather failure", "group": {"name": "opp-tests", "priority": 2}},
+      {"step": "acm-inspector", "failure_type": "pod_failure", "classification": "ACM Inspector failure", "group": {"name": "opp-tests", "priority": 2}}
+    ]
```

```suggestion
    ],
  "success_rules":
    [
      {"jira_project": "CSPIT", "jira_additional_labels": ["!default"]}
    ]
```
Should this also have success rules?